docs/tre-admins/auth.md

```diff
@@ -9,7 +9,8 @@ The following values are needed to be in place to run the script. (`/templates/c
 | Key | Description |
 | ----------- | ----------- |
 |TRE_ID|This is needed to build up the redirect URI for the Swagger App|
-|AAD_TENANT_ID|The tenant id of where your AAD identities will be placed. This can be different to the tenant where your Azure resources are created.
+|AAD_TENANT_ID|The tenant id of where your AAD identities will be placed. This can be different to the tenant where your Azure resources are created.|
+|AUTO_WORKSPACE_APP_REGISTRATION| Default of `false`. Setting this to true grants the `Application.ReadWrite.All` permission to the API AAD Application. This allows it to create other AAD applications, e.g. Workspaces.
```

Comment:
IT doesn't actually grant it does it? Does it enable the script to grant it or the TF to use it?

Reply:
This value is a top level flag that will be used by multiple pieces so that this feature can work. In this PR, the flag will be checked by the aad-app-reg.sh script and just enable a permission on the aad app. (as the text says)

It doesn't create workspace apps, it just says "the api app is able to create workspace apps" when required.